### PR TITLE
update token balance values

### PR DESCRIPTION
**Pull request by jennypollack**
this PR breaks down the total token value into value by balance and supply amount, which will be useful in the UI to help show uncollected drop/tin

currently, these values are only updated once/day - another PR will be needed to update the values more frequently

### CODE REVIEW

File: schema.graphql [hunk 1]
```diff
@@ -113,8 +113,10 @@ type Token @entity {
 type TokenBalance @entity {
   id: ID! # account address + token address
   owner: Account!
-  balance: BigInt!
-  value: BigInt!
+  balanceAmount: BigInt!
+  balanceValue: BigInt!
+  totalValue: BigInt!
+  tokenPrice: BigInt!
```
Comment (philipstanislaus) [inline schema.graphql:119]: Can we move the price into the `Token` instead?

Also, can we add `totalAmount`?

**Review by philipstanislaus — APPROVED** [review]
LGTM!

File: src/mappings/Tranche.ts [hunk 2]
```diff
@@ -33,7 +31,7 @@ export function handleSupplyOrder(call: SupplyOrderCall): void {
   tk.owners = pushUnique(tk.owners, account)
   tk.save()
 
-  let tokenBalance = loadOrCreateTokenBalance(account, token)
-  tokenBalance.pendingSupplyCurrency = amount
-  tokenBalance.save()
+  let tb = loadOrCreateTokenBalance(account, token)
+  calculateDisburse(tb, <PoolAddresses>poolAddresses)
```
Comment (jennypollack) [inline src/mappings/Tranche.ts:35]: cc @lucasvo can you clarify here if calling calcDisburse is functionally the same as setting the pendingSupplyCurrency to the amount

**Review by philipstanislaus — APPROVED** [review]
LGTM, thanks 🎉

**Pull request merged**